image-rs/Cargo.toml

```diff
@@ -39,7 +39,7 @@ serde = { workspace = true, features = ["serde_derive", "rc"] }
 serde_json.workspace = true
 serde_yaml = { version = "0.9", optional = true }
 sha2.workspace = true
-sigstore = { git = "https://github.com/sigstore/sigstore-rs.git", rev = "d5ba303", default-features = false, optional = true }
+sigstore = { git = "https://github.com/Xynnn007/sigstore-rs.git", rev = "232b337", default-features = false, optional = true }
```

Comment:
I understand that a fork is being used to a linting issue with the 1.72 rust toolchain (https://github.com/Xynnn007/sigstore-rs/commit/232b33720067337618187fe3c0b58c56312475e6). I think that a better approach is for the kata-agent build to use a newer version, and we are discussing this on the Kata AC meeting later today, so we should be able to switch to the 0.9.0 release if that goes well